subworkflows/nf-core/bam_ngscheckmate/main.nf

```diff
@@ -0,0 +1,49 @@
+include { BCFTOOLS_MPILEUP } from '../../../modules/nf-core/bcftools/mpileup/main'
+include { NGSCHECKMATE_NCM } from '../../../modules/nf-core/ngscheckmate/ncm/main'
+
+workflow BAM_NGSCHECKMATE {
+
+    take:
+    ch_input            // channel: [ val(meta1), bam/cram ]
+    ch_snp_bed          // channel: [ val(meta2), bed ]
+    ch_fasta            // channel: [ val(meta3), fasta ]
+
+    main:
+
+    ch_versions = Channel.empty()
+
+    ch_input_bed = ch_input.combine(ch_snp_bed.collect())
+                        // do something to combine the metas?
+                        .map{ input_meta, input_file, bed_meta, bed_file ->
+                            [input_meta, input_file, bed_file]
+                        }
+
+    BCFTOOLS_MPILEUP (ch_input_bed, ch_fasta.collect(), false)
+    ch_versions = ch_versions.mix(BCFTOOLS_MPILEUP.out.versions)
+
+    BCFTOOLS_MPILEUP
+    .out
```

Comment:
Indent the ops (here and below)?

Reply:
This is in the nf-core subworkflow, but I'll try and remember to do it when I convert to nf-test.